app/views/home/tos_faq.html.erb

```diff
@@ -221,7 +221,7 @@
 			<h5 class="heading" id="why_18">Why is "underage" defined as "under 18"?</h5>
 			<p>Though there is no international consensus, there is a trend to focus on 18 as an important age in regulating depictions of sexual activity (as opposed to actual sexual activity/age of consent, which is regulated in many more varied ways).  Thus, we decided that 18 would be helpful for the maximum number of users, including audiences as well as creators, though we recognize that no solution is perfect for everyone.  We encourage creators and recommenders to be more specific in tags or summaries where this would be useful to potential audiences.
 			</p>
-			<p><em>Note from the Content Policy committee:</em> With the exception of user icons, the Archive hosts only text, but we do plan to expand over time, and we do allow embeds of certain types of files <a href="https://archiveofourown.org/archive_faqs/8">hosted elsewhere</a>, which are also subject to the Content Policy.  Because regulations of sexually explicit content are generally concerned with visual depictions, there is potentially more flexibility for textual depictions.  The current rule is for 18 across the board, but we welcome suggestions on alternatives, especially from people with an interest in fan art.
+			<p><em>Note from the Content Policy committee:</em> With the exception of user icons, the Archive hosts only text, but we do plan to expand over time, and we do allow embeds of certain types of files <a href="http://archiveofourown.org/faq/posting-and-editing#hostsites">hosted elsewhere</a>, which are also subject to the Content Policy.  Because regulations of sexually explicit content are generally concerned with visual depictions, there is potentially more flexibility for textual depictions.  The current rule is for 18 across the board, but we welcome suggestions on alternatives, especially from people with an interest in fan art.
```

Comment:
Instead of hard-coding the beta site link (which has the same result as https://otwarchive.atlassian.net/browse/AO3-4935), you should use the Rails path, like `archive_faq_path("posting-and-editing", anchor: "hostsites")`.